tests/integration/pilot/tunneling/external-app/service-entry.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: external-app
+spec:
+  hosts:
+  - external-app.testdomain
```

Comment:
External app already exists, no need to recreate it: https://github.com/istio/istio/blob/a37782af0cd8282868271e2a05eb98e00864ab39/pkg/test/framework/components/echo/common/deployment/echos.go#L134

Reply:
Done.